app/views/pages/numbers.html.haml

```diff
@@ -1,26 +1,491 @@
-= render partial: 'pages/content'
-.small-11.small-centered.medium-10.large-8
-  - unless numbers[:cardinal].length.zero?
+- cache "numbers-fragment" do
+  .typography.small-11.small-centered.medium-10.large-8
+    %h1 Numbers
+    .clearfix_left.column-block
+      .translation_video{"data-ratio" => "0.75", :style => "display: none;"}
+        %video.video-js.vjs-default-skin{:controls => "controls", :controlslist => "nodownload", :loop => "loop", :preload => "auto"}
+          %source{:src => "http://s3-ap-southeast-2.amazonaws.com/nzsl-assets/number-signs.mp4", :type => "video/mp4"}
+            %source{:src => "http://s3-ap-southeast-2.amazonaws.com/nzsl-assets/number-signs.webm", :type => "video/webm"}
+      %a.button.translation_button{:href => "javascript:void(0);"}
+        %i.fi-play>
+        Play this page in NZSL
+    .rule_after.ckeditor_content
+      %p
+        Many numbers have alternate or varying forms in NZSL. Variation in numbers is linked to age, region, and personal
+        preference of signers. New (Auslan) numbers were introduced into schools with Australasian Signed English in 1979,
+        which differed from the number signs used by earlier generations.
+      %p
+        NZSL has several systems that incorporate number in which a number sign has a distinct movement and/or location
+        including:
+      %ul
+        %li
+          %a{:href => "#cardinal"} Cardinal Numbers
+          (counting)
+        %li
+          %a{:href => "#ordinal"} Ordinal Numbers
+          (ordering: 1st)
+        %li
+          %a{:href => "#age"} Age
+        %li
+          %a{:href => "#time"} Time
+        %li
+          %a{:href => "#money"} Money
+        %li
+          %a{:href => "#fractions"} Fractions
+  .small-11.small-centered.medium-10.large-8
     .typography
       %h2#cardinal Cardinal Numbers
-    = render partial: 'static_pages/grid', locals: {group: numbers[:cardinal]}
-  - unless numbers[:ordinal].length.zero?
+    / with divs because of IE7
+    .typography
+      %ul.no-bullet
+        %li.small-6.medium-4.grid-stack.float-left.text-left
+          %a.drawing{:href => "#{sign_path("5655")}"}
```

Comment:
I guess this would be moving it back slightly in the original direction but could the sign path, image name, and alt for these get stored in a big ol hash and looped over instead?

Reply:
I created this version of the file backwards from the HTML in browser which is why everything is unrolled. This page won't change at all until (if) we do a re-design so I'm inclined to be lazy and leave it as it is.

Reply:
That said, I'm happy to change it if y'all want me to.